ops/framework.py

```diff
@@ -115,32 +124,16 @@ def restore(self, snapshot):
 class Event:
```

Comment:
I'm definitely +1 to renaming this to `EventDescriptor` in a future PR to avoid confusion.